app/controllers/todos_controller.rb

```diff
@@ -78,7 +77,6 @@ def set_todo
 
     # Never trust parameters from the scary internet, only allow the white list through.
     def todo_params
-      params.require(:todo).permit(:description, :owner, :user_id, :avatar, :nombre)
-      #binding.pry
+      params.require(:todo).permit(:description, :owner, :user_id, :avatar, :nombre, :estate)
```

Comment:
I strongly recommend to only write and use variables in English

Reply:
ok i got it